content/en/news/releases/1.6.x/announcing-1.6.x/upgrade-notes/index.md

```diff
@@ -0,0 +1,78 @@
+
+To check if there is any `v1alpha1` security policy in the cluster, run the following commands:
+
+{{< text bash >}}
+$ kubectl get policies.authentication.istio.io --all-namespaces
+$ kubectl get meshpolicies.authentication.istio.io --all-namespaces
+$ kubectl get rbacconfigs.rbac.istio.io --all-namespaces
+$ kubectl get clusterrbacconfigs.rbac.istio.io --all-namespaces
+$ kubectl get serviceroles.rbac.istio.io --all-namespaces
+$ kubectl get servicerolebindings.rbac.istio.io --all-namespaces
+{{< /text >}}
+
+To make sure not to accidentally apply any `v1alpha1` security policy in the future, remove the CRD of the `v1alpha1` security policy using the following commands:
+
+{{< text bash >}}
+$ kubectl delete crd policies.authentication.istio.io
+$ kubectl delete crd meshpolicies.authentication.istio.io
+$ kubectl delete crd rbacconfigs.rbac.istio.io
+$ kubectl delete crd clusterrbacconfigs.rbac.istio.io
+$ kubectl delete crd serviceroles.rbac.istio.io
+$ kubectl delete crd servicerolebindings.rbac.istio.io
+{{< /text >}}
+
+## Istio configuration during installation
+
+Historically, Istio deployed certain configuration objects as part of the installation. This caused problems with upgrades, a confusing user experience, and makes the installation less flexible. As a result, we minimized the configurations we ship as part of the installation.
+
+This includes a variety of configurations:
+
+* The `global.mtls.enabled` setting previously enabled strict mTLS. This setting was removed to avoid confusion.  You should instead configure a `PeerAuthentication` policy for [strict mTLS](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode)
+* The default `Gateway` object, and associated `Certificate` object, are no longer installed by default. See the [Ingress task](/docs/tasks/traffic-management/ingress/) for information on configuring a Gateway.
+* `Ingress` objects for telemetry addons are no longer created. See [Remotely Accessing Telemetry Addons](/docs/tasks/observability/gateways/) for more information on exposing these externally.
+* The default `Sidecar` configuration was previously defined with an automatically created `Sidecar` resource. This has been changed to an internal implementation detail and should have no visible impact.
+
+## Communicating with Istiod using External Workloads
+
+This release makes the Istiod host cluster-local by default. This means that the Istio control plane is not accessible to workloads that reside outside the cluster. It is now recommended that external workloads access Istiod via the ingress gateway. This change was needed in order to support multicluster master/remote configurations. Future releases will remove this limitation.
```

Comment:
Is reaching Istiod through a gateway required or just recommended? AFAIK, it's required. @brian-avery Please confirm with Networking WG.
```suggestion
Istio 1.6 configures Istiod to be `cluster-local` by default. This configuration makes the Istio control plane unreachable for workloads outside the cluster. Now, external workloads can access Istiod via an ingress gateway. To support multicluster deployments with master and remote clusters, the default configuration needed to change. The Istio team is actively working to remove the gateway requirement.
```

Reply:
@nmittler @rcaballeromx Please take a look at this: 

```suggestion
Istio 1.6 configures Istiod to be `cluster-local` by default. This change means that an Istio gateway will only send traffic to an Istiod instance located in the same cluster. Workloads external to the cluster must now access Istiod via the ingress gateway. To support multicluster deployments with master and remote clusters, the default configuration needed to change. The Istio team is actively working to remove the gateway requirement.
```

Reply:
This sentence is not correct: "The Istio team is actively working to remove the gateway requirement". Gateway will continue to be the recommendation on istio.io going forward. The alternative is ILB which is cloud vendor-specific and can't be documented under install.

So the limitation of cluster-local is what we *may* remove going forward.

Reply:
Thanks @nmittler. Updated.